src/core/pick-models.js

```diff
@@ -59,7 +59,7 @@ export default function pickModels(gl, {
         format: gl.RGBA,
         type: gl.UNSIGNED_BYTE});
 
-      const isPicked = color[0] !== 0 || color[1] !== 0 || color[2] !== 0 || color[3] !== 0;
+      const isPicked = color[0] !== 0 || color[1] !== 0 || color[2] !== 0;
```

Comment:
@ibgreen , when we write picking color to FB , alpha channel is always 1.0, hence removed checking `color[3] !== 0`, verified picking in Instancing example still works.